iothub/device/src/Transport/Mqtt/MqttTransportSettings.cs

```diff
@@ -250,5 +250,21 @@ public TransportType GetTransportType()
         /// Used by Edge runtime to specify an authentication chain for Edge-to-Edge connections.
         /// </summary>
         internal string AuthenticationChain { get; set; }
+
+        /// <summary>
+        /// The DotNetty event loop GracefulShutdown timeout.
+        /// </summary>
+        /// <remarks>
+        /// The DotNetty Mqtt transport has a graceful shutdown of the event loop that can block the <see cref="DeviceClient.CloseAsync(System.Threading.CancellationToken)"/> method. This timeout allows you to configure how long the shutdown event will wait for before forcefully shutting down the event loop. Change this if you need your application to respond to the CloseAsync command faster.
+        /// </remarks>
+        public TimeSpan GracefulEventLoopShutdownTimeout { get; set; } = TimeSpan.FromSeconds(1);
+
+        /// <summary>
+        /// Sets the MqttTransport to use a single Event Loop Group.
+        /// </summary>
+        /// <remarks>
+        /// This setting is to retain legacy behavior that will use a single Event Loop Group which is the thread pool model for DotNetty. If this is set to false a new group will be created per instance of the Device Client.
```

Comment:
We have 1 instance of mqtt adapter per device client, and we can have 1 instance of device client for a particular device identity. Considering that, what does this mean:
"If this is set to false a new group will be created per instance of the Device Client.", i.e. does this refer to the scenario where we dispose a client instance and reinitialize?